code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -421,7 +421,7 @@
 
 /obj/item/weapon/storage/belt/sabre
 	name = "sabre sheath"
-	desc = "An ornate sheath designed to hold an officer's blade."
+	desc = "An ornate sheath designed to hold an officer's blade. Alt-Click to quickly draw the sheathed blade."

```

Comment:
we really need to standardize this kind of thing, i tried to do something like that in https://github.com/tgstation/tgstation/pull/17050 but got lazy, please do something like that thanks